.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
 * @launchdarkly/team-core-product-frontend
-/packages/icons/src @matthewferry @antonio-darkly
+/packages/icons/src @matthewferry @antonio-darkly @launchdarkly/team-core-product-frontend
```

Need to add changeset so PRs for icons can be merged with designer approval

```suggestion
/packages/icons/src @launchdarkly/team-core-product-frontend @matthewferry @antonio-darkly 
/.changeset @launchdarkly/team-core-product-frontend @matthewferry @antonio-darkly 
```